GrandCentralBoard/Widgets/Slack/SlackMessage.swift

```diff
@@ -10,4 +10,9 @@ import Foundation
 
 struct SlackMessage {
     let text: String
+    let timestamp: NSDate
+
+    var channel: String?
+    var author: String?
```

Comment:
Yes, it was optional because it is an optional field in SlackKit. But I did add a guard before creation of SlackMessage.